### ISSUE TITLE
Not found: Op type not registered 'LeakyRelu'

### ISSUE THREAD

**Issue by tienduchoang** [YunYang1994/tensorflow-yolov3#68]
Hello YunYang1994, I try to deploy yolov3 using tensorrt inference servers. Thank for your code, i converted yolo -> tensorflow (.pb). But i got this error. can u help me?
Thanks

cannot be loaded: Not found: Op type not registered 'LeakyRelu' in binary running on 2a46b9c85901. Make sure the Op and Kernel are registered in the binary running in this process. Note that if you are loading a saved graph which used ops from tf.contrib, accessing (e.g.) `tf.contrib.resampler` should be done before importing the graph, as contrib ops are lazily registered when the module is first accessed.

**Comment by tienduchoang** [comment]
i got this error because I use tf version 1.13. downgrade your tensorflow version to 1.9.0

**Comment by sifeng666** [comment]
Hi tienduchoang, I'm wondering how to convert this into TensorRT, could you tell me where I should change the codes? Thank you!

**Comment by zhaopengpeng8866** [comment]
I also encountered the same problem, turning yolo3's checkpoint model into a .pb model, and then calling the .pb model in the c++ environment compiled with tensorflow, I got an error: 
Not found: Op type not registered 'LeakyRelu' in binary running on LAPTOP-JNCE3E16. Make sure the Op and Kernel are registered in the binary running in this process.
I also tried using other compiled versions of tensorflow, such as 1.7, 1.8, 1.9, and 1.10, etc., which were not resolved.
